src/scss/_footer.scss

```diff
@@ -104,30 +104,34 @@ footer {
 }
 
 @media #{$phone-media-query} {
+  footer {
+    padding-top: 60px;
```

Comment:
I added 60px of padding over the footer only on mobile because there was the thank you message blocking the input field of the newsletter.